models/src_snapchat.yml

```diff
@@ -319,3 +331,150 @@ sources:
+            description: The number of milli seconds spent watching your campaign across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
+          - name: video_views
+            description: The number of times your video was watched at least 2 seconds or swiped up, whichever comes first
+          - name: view_completion
+            description: The number of times your video campaign was viewed to completion (97% )
+          - name: view_time_millis
+            description: The number of milli seconds spent watching your campaign across all users reached.
+          - name: impressions
+            description: The number of impressions for a campaign.
+          - name: spend
+            description: The amount of spend in microdollars for a campaign.
+          - name: swipes
+            description: The number of swipes for a campaign. Swipes are when your ad was swiped up on or the CTA was tapped to view the attachment below.
+          - name: conversion_purchases_value
+            description: The total value of purchases attributed to a campaign (microcurrency in Ad Account's currency).
+          - name: conversion_purchases
+            description: The total number of purchases attributed to a campaign.
+          - name: conversion_add_billing
+            description: Total number of attributed "ADD_BILLING" conversion events for a campaign in the hour of the record.
+          - name: conversion_save
+            description: Total number of attributed "SAVE" conversion events for a campaign in the hour of the record.
+          - name: conversion_add_cart
+            description: Total number of attributed "ADD_CART" conversion events for a campaign in the hour of the record.
+          - name: conversion_sign_ups 
+            description: Total number of attributed "SIGN_UP" conversion events for a campaign in the hour of the record.
+          - name: conversion_start_checkout
+            description: Total number of attributed "START_CHECKOUT" conversion events for a campaign in the hour of the record.
+          - name: conversion_searches
+            description: Total number of attributed "SEARCH" conversion events for a campaign in the hour of the record.
+          - name: conversion_view_content
+            description: Total number of attributed "VIEW_CONTENT" conversion events for a campaign in the hour of the record.
+          - name: conversion_app_opens
+            description: Total number of attributed "APP_OPEN" conversion events for a campaign in the hour of the record.
+          - name: conversion_level_completes
+            description: Total number of attributed "LEVEL_COMPLETE" conversion events for a campaign in the hour of the record.
+          - name: conversion_page_views
+            description: Total number of attributed "PAGE_VIEW" conversion events for a campaign in the hour of the record.
```

Comment:
These fields aren't present in the `get_*_columns`  macro, so I assume these are passthrough columns. Can we document that these are passthrough fields and [add a link to the README?](https://github.com/fivetran/dbt_snapchat_ads_source/blob/main/README.md#passing-through-additional-metrics). [You can see here ](https://github.com/fivetran/dbt_facebook_ads_source/pull/44/files#diff-91655d3722ecb28d559b02d3ab2da5f6a7f7675de7cec8624c9e39d80686776fR25-R34)for an example.

Reply:
FYI these fields are brought in via the conversion_fields variable instead of passthrough columns, so I'd use this [link](https://github.com/fivetran/dbt_snapchat_ads_source/blob/main/README.md#configuring-conversion-fields) instead

Reply:
Ahh thanks @fivetran-jamie!

Reply:
updated!